kong/router/utils.lua

```diff
@@ -104,8 +104,14 @@ local function check_select_params(req_method, req_uri, req_host, req_scheme,
 end
 
 
-local function add_debug_headers(var, header, match_t)
-  if not var.http_kong_debug then
+local get_header
+if ngx.config.subsystem == "http" then
+  get_header = require("kong.tools.http").get_header
+end
+
+
+local function add_debug_headers(ctx, header, match_t)
```

Comment:
We should not change the param `var` to `ctx`, because `var` is used for unit-testing's mock_ngx.

Reply:
we canfix the tests

Reply:
test is fixed now.